react/training-project/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import actionCreators from '~redux/Auth/actions';
+
+import LoginForm from './layout';
+
+/* eslint-disable no-func-assign */
+class LoginPage extends Component {
+  handleSubmit = values => {
```

Comment:
Try to be consistent. For one-line methods you're using two syntax; sometimes you use this one:
```
handleSubmit = values => {
  this.props.logIn(values);
}
```
And sometimes you use this one:
`
handleSubmit = values => this.props.logIn(values);
`